src/app/applications/datatarget/datatarget-test-connection/datatarget-test-connection.component.html

```diff
@@ -0,0 +1,82 @@
+                  class="mt-3 btn btn-primary">
+            {{ "DATATARGET.TEST_CONNECTION.TEST-CONNECTION" | translate }}
+          </button>
+          <p class="mt-2">{{ "DATATARGET.TEST_CONNECTION.SEND-PING-TO-DATATARGET" | translate }}</p>
+          <mat-divider></mat-divider>
+        </div>
+        <p class="mt-3 font-weight-bold">{{ "DATATARGET.TEST_CONNECTION.SEND-LATEST-PACKAGE" | translate }}</p>
+        <mat-select
+          (selectionChange)="getLastPackage($event)"
+          [placeholder]="'DATATARGET.TEST_CONNECTION.CHOOSE-IOT-DEVICE' | translate"
+          class="mt-3 form-control"
+        >
+          <mat-option *ngFor="let iotDevice of iotDevices" [value]="iotDevice.id">
+            {{ iotDevice.name }}
+          </mat-option>
+        </mat-select>
+        <mat-select
+          [(value)]="payloadDecoderId"
+          [placeholder]="'DATATARGET.TEST_CONNECTION.CHOOSE-PAYLOAD-DECODER' | translate"
+          class="mt-3 form-control">
+          <mat-option [value]="0">
+            {{ "QUESTION.DATATARGET.NO-PAYLOAD-DECODER-SELECTED" | translate }}
+          </mat-option>
+          <mat-option *ngFor="let payloadDecoder of payloadDecoders" [value]="payloadDecoder.id">
+            {{ payloadDecoder.name }}
+          </mat-option>
+        </mat-select>
+        <button (click)="pingDataTarget(payloadData)" [disabled]="!payloadData"
+                class="mt-3 btn btn-primary">{{ "DATATARGET.TEST_CONNECTION.SEND-DATA-PACKAGE" | translate }}
+        </button>
+        <p class="mt-2" style="white-space: pre-wrap">
+          {{ "DATATARGET.TEST_CONNECTION.SEND-DATA-PACKAGE-DESCRIPTION" | translate }}
+          {{ dataTarget.type === DataTargetType.MQTT ? ("DATATARGET.TEST_CONNECTION.MQTT-DISCLAIMER"| translate) : "" }}
+        </p>
+      </div>
+    </div>
+    <div class="col-md-6 d-flex align-items-stretch">
+      <div class="jumbotron jumbotron--full-width">
+        <h3>{{ "DATATARGET.TEST_CONNECTION.CHOSEN-DATA-PACKAGE" | translate }}</h3>
+        <ngx-monaco-editor
+          [(ngModel)]="payloadData"
+          [ngModelOptions]="{ standalone: true }"
+          [options]="editorJsonOutpuOptions"
+          style="height: 85%"
+        >
+        </ngx-monaco-editor>
+      </div>
+    </div>
+    <div class="col-md-6 d-flex align-items-stretch">
+      <div class="jumbotron jumbotron--full-width">
+        <h3>{{ "DATATARGET.TEST_CONNECTION.SENT-DECODED-REQUEST" | translate }}</h3>
+        <ngx-monaco-editor
+          [(ngModel)]="decodedData"
+          [ngModelOptions]="{ standalone: true }"
+          [options]="editorJsonOutpuOptions"
+        >
+        </ngx-monaco-editor>
+      </div>
+    </div>
+    <div class="col-md-6 d-flex align-items-stretch">
+      <div class="jumbotron jumbotron--full-width">
+        <h3>{{ "DATATARGET.TEST_CONNECTION.RECEIVED-RESPONSE" | translate }}</h3>
+        <div *ngIf="loading">
+          <app-loading-spinner></app-loading-spinner>
```

Comment:
Måske man skulle overveje at placere spinneren et andet sted, Fx i midten af skærmen eller øverst. Det ser lidt spøjst ud, med de to JSON views som bliver rykket lidt op og ned med nuværende styling, når spinneren aktiveres

Reply:
Har flyttet den neders så er der ikke nogen komponenter der flytter sig